difftest/dpi_t1rocketemu/src/drive.rs

```diff
@@ -249,29 +245,24 @@ impl Driver {
   pub(crate) fn watchdog(&mut self) -> u8 {
     const WATCHDOG_CONTINUE: u8 = 0;
     const WATCHDOG_TIMEOUT: u8 = 1;
+    const WATCHDOG_QUIT: u8 = 255;
```

Comment:
Since we have the MMIO write infra, I personally prefer using a special MMIO write as exit function.

Reply:
Then watchdog is only for timeout.

Reply:
The name watchdog is a bit confusing, maybe a better name is `t1_cosim_get_status`.

A special MMIO write can not reduce the complexity here. Note that we can not immediate call `$finish` during the mmio write DPI function, because we must wait TestBench becoming idle. If we do not poll rust side every cycle, an alternative design is expose a DPI `set_finish_flag` to rust side. Personally I don't like DPI calls jumping back and forth, so I tend to poll every cycle here.